src/test/scala-2.11/markets/clearing/engines/CDAMatchingEngineSpec.scala

```diff
@@ -469,11 +468,13 @@ class CDAMatchingEngineSpec extends TestKit(ActorSystem("CDAMatchingEngineSpec")
 
       Then("the matching engine should generate a PartialMatch at the best limit price.")
 
-      val partialFill = PartialMatch(marketBidOrder, askOrder, bidPrice)
-
       val residualAskQuantity = askQuantity - marketBidQuantity
-      val residualAskOrder = askOrder.split(residualAskQuantity)
-      val totalFill = TotalMatch(limitBidOrder, residualAskOrder, bidPrice)
+      val (filledAskOrder, residualAskOrder) = askOrder.split(residualAskQuantity)
+      val partialFill = PartialMatch(marketBidOrder, filledAskOrder, bidPrice)
+
+      val residualBidQuantity = limitBidQuantity - residualAskQuantity
+      val (filledBidOrder, residualBidOrder) = limitBidOrder.split(residualBidQuantity)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Field name does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1539841712/issues/source?bid=2921361&fileBranchId=3014278#l476)